### PR TITLE
Add policy link. DMR-64

### PR DESCRIPTION
**Pull request by VivianChu**

### CODE REVIEW

File: coverage/.resultset.json [hunk 1]
```diff
@@ -1,18 +1,483 @@
 {
   "RSpec": {
     "coverage": {
-      "/Users/Vivian_Chu/git/dmr/app/helpers/application_helper.rb": [
+      "/Users/Vivian_Chu/git/dmr/app/controllers/application_controller.rb": [
```
Comment (mcritchlow) [inline coverage/.resultset.json]: do these files need to be in version control? If not, can you delete them in this PR?

**Comment by VivianChu** [comment]
@mcritchlow  The files have been removed.  Thanks

**Comment by mcritchlow** [comment]
looks good! would you mind squashing the commits to one when you have a chance? then I'll get this merged.

**Comment by VivianChu** [comment]
The commits have been squashed.

**Pull request merged**